src/coreclr/vm/callingconvention.h

```diff
@@ -1874,11 +1874,14 @@ int ArgIteratorTemplate<ARGITERATOR_BASE>::GetNextOffset()
 
     return argOfs;
 #elif defined(TARGET_WASM)
-    int cbArg = ALIGN_UP(StackElemSize(argSize), TARGET_POINTER_SIZE);
+    // See duplicate definition in interpreter code.
+#define INTERP_STACK_SLOT_SIZE 8
```

Comment:
I'd like to see a comment in interpretershared mentioning that callingconvention.h needs to be updated if the define is changed

Reply:
I've never found those comments very helpful. Instead I've used the exact same define name and indicated it is a duplicate. The first steps for these things are typically, "where is this used?", which should bring people here.

Reply:
I was able to refactor this and avoid redeclaring the define.